throughput/src/bin/z_put_thr.rs

```diff
@@ -15,26 +15,22 @@ use async_std::{sync::Arc, task};
 use clap::Parser;
 use std::{
     path::PathBuf,
-    str::FromStr,
     sync::atomic::{AtomicUsize, Ordering},
     time::Duration,
 };
-use zenoh::{
-    config::{whatami::WhatAmI, Config},
-    prelude::{Locator, Value},
-    publication::CongestionControl,
-};
+use zenoh::{config::Config, prelude::Value};
+use zenoh_protocol_core::{CongestionControl, EndPoint, WhatAmI};
 
 #[derive(Debug, Parser)]
 #[clap(name = "z_put_thr")]
 struct Opt {
     /// locator(s), e.g. --locator tcp/127.0.0.1:7447,tcp/127.0.0.1:7448
     #[clap(short, long, value_delimiter = ',')]
-    locator: Vec<Locator>,
+    locator: Vec<EndPoint>,
```

Comment:
@YuanYuYuan What do you think?

Reply:
Personally speaking, I would choose `endpoint` > `locator` > `connect`.  I don't prefer `connect` because it's used for peer mode, and `listen` is for router/client. The remaining problem is we need to change the command line flags specified in testing scripts. 

![image](https://user-images.githubusercontent.com/10692887/159223711-a690d557-dc50-4444-a608-ec6b60087d3a.png)

Reply:
I'm okay with that. Changing the command line flags in zenoh-tests should not be a problem at this moment. According to the above comments, I will also vote to change to `endpoint`.

Reply:
Okay, then let's change all the arguments `locator` to `endpoint`.